final_prep/mod_zero_hero.rb

```diff
@@ -1,84 +1,155 @@
+  if danger_level >= 50
+    puts bad_excuse
+  elsif danger_level == (10..50)
+    put save_the_day
+  elsif danger_level <= 10
+    puts "Meh, hard pass"
+  end
+end
 
 # Your function should include an if/else statement that meets the following criteria
 #   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the bad_excuse to the terminal
 #   - Anything danger_level that is between 10 and 50 should result in printing the save_the_day string to the terminal
 #   - If the danger_level is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the terminal.
 
-#Test Cases
+
+#Test Cases *** i just got to the test cases at 30 minutes
+
 announcement = 'Never fear, the Courageous Curly Bracket is here!'
 excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
 # assess_situation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
 #assess_situation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
 #assess_situation(3, announcement, excuse) > should print - "Meh. Hard pass."
 
+assess_situation(99, announcement, excuse)
+assess_situation(21, announcement, excuse)
+assess_situation(3, announcement, excuse)
+
 # Declare a new variable - scary_monster - assigned to an hash with the following key/values
 #   - name (string)
 #   - smell (string)
 #   - weight (integer)
 #   - citiesDestroyed (array)
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
-
+scary_monster = {
+  name: "Tobby",
+  smell: "dandelion",
+  weight: 250,
+  cities_destroyed: ["New York", "Atlanta", "Baltimore"],
+  lucky_numbers: [32, 78, 1],
+  address: {
+    number: 1215,
+    street: "Perrine St",
+    state: "Montana",
+    zip: 00023,
+  }
+}
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
 
 # - Create the following class methods
 #   - say_name, should print the hero's name to the terminal
 #   - maximize_energy, should update the energy_level to 1000
 #   - gain_power, should take an argument of a number and INCREASE the power_level by that number
 
 # - Create 2 instances of your SuperHero class
-
+class SuperHero
+  attr_accessor :name, :super_power, :age, :arch_nemesis, :power_level, :energy_level
+  def initialize (name, super_power, age)
+    @name = name
+    @super_power = super_power
+    @age = age
+    @arch_nemesis = "The Syntax Error"
+    @power_level = 100
+    @energy_level = 50
+  end
+
+  def say_name(name)
```

Comment:
I don't think `say_name` needs to accept an argument here, you already have access to the hero's name in `@name`.